src/model/channel/message.rs

```diff
@@ -72,6 +72,12 @@ pub struct Message {
     /// Array of [`Role`]s' Ids mentioned in the message.
     pub mention_roles: Vec<RoleId>,
     /// Channels specifically mentioned in this message.
+    /// Please note:
```

Comment:
Minor wording and formatting change to be consistent with other notes in documentation.

```suggestion
    ///
    /// **Note**:
```